src/infrastructure/validation/logger/FileLogger.java

```diff
@@ -65,11 +65,11 @@ protected FileLogger(List<LogLevel> enabledLogLevelsList, boolean enableTestMode
 		
 		try {
 			String home = System.getProperty("user.home");
-			logFilePath = home+"/.config/";
+			logFilePath = home+"/.config/KoraKaagaz";
 		} catch (SecurityException se) {
```

Comment:
Maybe in a folder called logs under KoraKaagaz. Can be more intuitive to search.

Reply:
okay
changing it to `$HOME/.config/KoraKaagaz/logs/`